benches/benches/set/crypto.rs

```diff
@@ -20,9 +20,9 @@ pub fn run(c: &mut Criterion) {
     let signature = Signature::sign(&secret, &message);
 
     run_group_ref(
```

Comment:
should we also setup benchmarks for ecr1 and ed19?

Reply:
There is an issue for that=) https://github.com/FuelLabs/fuel-core/issues/1254